src/__tests__/relative.spec.ts

```diff
@@ -0,0 +1,8 @@
+import { relative } from '../';
+
+describe('relative', () => {
+  it('handles URLs', () => {
```

Comment:
Should also add some tests to make sure it handles the various file path cases (posix, windows, forward slash, mixed slash, etc).